### PR TITLE
image-pruning: Improve help and error reporting

### PR DESCRIPTION
**Pull request by miminar**
Resolves: [bz#1469654](https://bugzilla.redhat.com/show_bug.cgi?id=1469654)

### CODE REVIEW

File: pkg/oc/admin/prune/images.go [hunk 1]
```diff
@@ -48,7 +51,9 @@ var (
 		registry by supplying --all=false flag.
 
 		By default, the prune operation performs a dry run making no changes to internal registry. A
-		--confirm flag is needed for changes to be effective.
+		--confirm flag is needed for changes to be effective. The flag requires a valid route to the
+		integrated Docker registry. If this command is run outside of the cluster, the route needs
+		to be provided using --registry-url.
```
Comment (dmage) [inline pkg/oc/admin/prune/images.go]: Sorry, but what does "run outside of the cluster" mean?

Comment (miminar) [reply pkg/oc/admin/prune/images.go]: **Inside** of the cluster means: from one of the nodes belonging to the OpenShift cluster with DNS managed either by master or by some internal DNS server.

Outside means: from anywhere else - and probably with NAT in the middle. 

Is there any less-confusing short description I could use?

File: pkg/oc/admin/prune/images.go [hunk 2]
```diff
@@ -139,7 +144,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
 		},
 	}
 
-	cmd.Flags().BoolVar(&opts.Confirm, "confirm", opts.Confirm, "If true, specify that image pruning should proceed. Defaults to false, displaying what would be deleted but not actually deleting anything.")
+	cmd.Flags().BoolVar(&opts.Confirm, "confirm", opts.Confirm, "If true, specify that image pruning should proceed. Defaults to false, displaying what would be deleted but not actually deleting anything. Requires a valid route to the integrated Docker registry.")
```
Comment (juanvallejo) [inline pkg/oc/admin/prune/images.go]: Maybe also explain here that a valid route can be provided with --registry-url

File: pkg/oc/admin/prune/images.go (hunk 1 above)
Comment (dmage) [reply pkg/oc/admin/prune/images.go]: Oh, so `oc` should be able to reach the integrated registry either by internal IP or by external address from --registry-url. I see.

File: pkg/oc/admin/prune/images.go (hunk 2 above)
Comment (miminar) [reply pkg/oc/admin/prune/images.go]: @juanvallejo I added only a reference to the other flag to keep the message short.

File: pkg/oc/admin/prune/images.go (hunk 1 above)
Comment (bparees) [reply pkg/oc/admin/prune/images.go]: would "outside of the cluster network" make this clearer, perhaps?

**Comment by bparees** [comment]
one nit and lgtm.
/approve
/retest

File: pkg/oc/admin/prune/images.go (hunk 1 above)
Comment (miminar) [reply pkg/oc/admin/prune/images.go]: > would "outside of the cluster network" make this clearer, perhaps?

done

**Pull request merged**